src/main/java/frame/panels/TransactionsPanel.java

```diff
@@ -1,22 +1,28 @@
 package frame.panels;
 
+import com.github.lgooddatepicker.components.DatePicker;
+import com.github.lgooddatepicker.components.DatePickerSettings;
+
 import javax.swing.*;
 import javax.swing.table.DefaultTableModel;
 import java.awt.*;
 import java.awt.event.ActionEvent;
+import java.time.LocalDate;
 
 public class TransactionsPanel extends JPanel{
     private final String[] currencyTypes;
     private JPanel newTransactionPanel;
     private JTable transactionTable;
-    private JTextField transactionDateInput;
     private JTextField vendorInput;
     private JComboBox<String> transactionTypeInput;
     private JTextField quantityInput;
     private JTextField rateInput;
-    private JTextField maturityDateInput;
     private JComboBox<String> foreignCurrencyInput;
+    public JTextField transactionDateInput;
+    private final LocalDate first = LocalDate.of(2012, 1, 1);//confirm
```

Comment:
Confirmed in chat, comment can be removed

Reply:
oh i forgot to remove that - since we're doing the last 100 days im gonna get rid of the variable entirely; it's no longer being used